Makefile

```diff
@@ -130,7 +127,10 @@ jenkins_run:
 # Requiered packages for relaunched systems
 packages:
 	sudo apt-get update
-	sudo apt-get install libncurses5-dev texinfo autogen autoconf2.64 g++ libexpat1-dev flex bison gperf cmake libxml2-dev libtool zlib1g-dev libglib2.0-dev make pkg-config gawk subversion expect git libxml2-utils syslinux xsltproc yasm iasl lynx unzip qemu
+	sudo apt-get install libncurses5-dev texinfo autogen autoconf2.64 g++ libexpat1-dev \
+			     flex bison gperf cmake libxml2-dev libtool zlib1g-dev libglib2.0-dev \
+			     make pkg-config gawk subversion expect git libxml2-utils syslinux \
+			     xsltproc yasm iasl lynx unzip qemu
```

Comment:
what does this packages command do? It might be uncomplete, please remember that we wanted to integrate isc-dhcpd

Reply:
I don't know where this is used... @BugUser0815 ideas?

Reply:
This command is needed for developers who want to compile natively on their machines. They install all packages from this command and are good to go.
Maybe the Vagrant provision file could call the same command to install the required packages?
Consequently there would be no code duplicates any more.

Reply:
Made the necessary changes and added the two missing packages for tftp and dhcp server